File: Modul 7/Soal 1/PRAK601-2210817120012-Noorlaini Marratain.py
b, k, = map(int, input().split())
angka = []
a = 0
isi = list(map(int, input().split()))
print()
for i in range(b):
    angka.append(isi[a : a + k])
    a += k
for b in angka:
    for isi in b:
        print(isi, end=' ')
    print()
print()

File: Modul 7/Soal 1/PRAK601-2210817120012-Noorlaini Marratain.c
#include <stdio.h>

int main()
{
    int b, k, isi;
    scanf("%d %d", &b, &k);
    int angka[b][k];
    for(int i=0; i<b; i++){
        for(int j=0; j<k; j++){
            scanf("%d", &isi);
            angka[i][j] = isi;
        }
    }
    printf("\n");
    for(int i=0; i<b; i++){
        for(int j=0; j<k; j++){
            printf("%d ", angka[i][j]);
        }
        printf("\n");
    }
    return 0;
}
